libr/io/p/io_ptrace.c

```diff
@@ -220,14 +226,15 @@ static RIODesc *__open(RIO *io, const char *file, int rw, int mode) {
 			case EPERM:
 				ret = pid;
 				eprintf ("ptrace_attach: Operation not permitted\n");
-				break;
+				return NULL;
 			case EINVAL:
 				perror ("ptrace: Cannot attach");
 				eprintf ("ERRNO: %d (EINVAL)\n", errno);
-				break;
+				return NULL;
+			default:
```

Comment:
Just return null once if ebusy is not ignores now.

Also. Is your patch working? Did u tested it?

Reply:
> Just return null once if ebusy is not ignores now.
wat

> Also. Is your patch working? Did u tested it?
yes